.ci/openshift-ci/run_smoke_test.sh

```diff
@@ -55,12 +67,11 @@ else
 	info "HTTP server is unreachable"
 fi
 
-info "Deleting resources created"
-oc delete -f ${script_dir}/smoke/service.yaml
-
-# Delete the pod.
+# Delete the resources.
 #
-info "Deleting the ${pod} pod"
+info "Deleting the resources"
+[ "$is_ocp" -eq 1 ] && oc delete -f ${script_dir}/smoke/service.yaml
+[ "$is_ocp" -eq 0 ] && oc delete -f ${script_dir}/smoke/service_kubernetes.yaml
```

Comment:
I'd personally express that with an `if then else`. Matter of taste :wink:

Reply:
ack, again, remnant from previsou version which handled things differently